### PR TITLE
Upgrade scaffold and introduce Github Action

### PR DESCRIPTION
**Pull request by sttk**
I modified this project referring to [replace-ext](https://github.com/gulpjs/replace-ext).

### CODE REVIEW

File: .github/security.md [hunk 1]
```diff
@@ -1,4 +1,15 @@
-## Security contact information
+# Security Policy
```
Comment (phated) [inline .github/security.md:1]: @sttk I added a "global" security policy to the gulpjs github organization, so we can probably remove this.

File: README.md [hunk 2]
```diff
@@ -62,22 +60,16 @@ If a loader is already registered, this will simply return `true`.
...
 MIT
 
-[interpret]: http://github.com/gulpjs/interpret
-[Liftoff]: http://github.com/gulpjs/liftoff
-
-[downloads-image]: http://img.shields.io/npm/dm/rechoir.svg
+<!-- prettier-ignore-start -->
+[interpret]: https://github.com/gulpjs/interpret
+[require.extensions]: https://nodejs.org/api/modules.html#modules_require_extensions
+[liftoff]: https://github.com/js-cli/js-liftoff
```
Comment (phated) [inline README.md:66]: I've been trying to move these to the bottom of the URL block so the upgrade tool has an easier time diffing. They could even exist in their own `<!-- prettier-ignore-start -->` block at the bottom.

File: test/index.js [hunk 3]
```diff
@@ -252,6 +269,20 @@ describe('rechoir', function() {
 
       done();
     });
-  });
 
+    it('should register a module loader for the specified extension with cwd', function(done) {
```
Comment (phated) [inline test/index.js:273]: New test?

File: .github/security.md (hunk 1 above)
Comment (sttk) [reply .github/security.md:1]: OK. I'll remove this file.

File: README.md (hunk 2 above)
Comment (sttk) [reply README.md:66]: Got it.

File: .github/security.md (hunk 1 above)
Comment (sttk) [reply .github/security.md:1]: Got it. I'll remove this file.

File: README.md (hunk 2 above)
Comment (sttk) [reply README.md:66]: Got it.

File: test/index.js (hunk 3 above)
Comment (sttk) [reply test/index.js:273]: Yes. there was no case about cwd option and this case make coverage to 100%.

**Comment by sttk** [comment]
@phated I'm sorry for too late reply. I modified what you pointed out.

**Comment by phated** [comment]
I'm going to make some more changes to the scaffold and apply to this repository. I will probably apply them to this PR and then merge it.

**Comment by phated** [comment]
Thanks @sttk! I'm going to be releasing 0.8.0 shortly. I appreciate your work here 🍻

**Pull request merged**